src/build.rs

```diff
@@ -166,7 +163,12 @@ impl BuildQueue {
         let (tx, rx) = channel();
         self.pending.lock().unwrap().push(tx);
         if priority == BuildPriority::Normal {
-            thread::sleep(Duration::from_millis(WAIT_TO_BUILD));
+            let wait_to_build = { // Release lock before we sleep
+                let config = self.config.lock().unwrap();
+                config.wait_to_build
+            };
+            
+            thread::sleep(Duration::from_millis(wait_to_build as u64));
```

Comment:
Sure. I need to add a new impl for ConfigType for that.
Should the variant name be "<unsigned 64bit integer>"?